core/src/avm2/globals/flash/display/DisplayObject.as

```diff
@@ -1,5 +1,6 @@
-package flash.display {
-    
+package flash.display
```

Comment:
The brackets in this file seem to have been moved to the next line each- to maintain formatting with the rest of the AS globals, they should remain on the previous line.